content/pages/equipment/hammer_dressing.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "How To: Dress a Hammer"
+description: "A guide to preparing your hammer for working metal"
+date: 2024-02-11T02:33:56Z
+tags:
+- howto
+---
+{{< notice warning >}}
+Always wear a respirator and eye protection when grinding metal
```

Comment:
I think mentioning that guards for angle grinders are *mandatory* would be good here (or below with the angle grinders)